src/components/common/ReserveAPR.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Tooltip, Typography, useTheme } from '@mui/material';
+import * as formatter from '../../utils/formatter';
+import { Icon } from './Icon';
+
+interface ReserveAprParams {
+  reserveSymbol: string;
+  reserveApr: number;
+  emissionApr: number | undefined;
+  isSupply: boolean;
+}
+
+export const ReserveApr = ({
+  reserveSymbol,
+  reserveApr,
+  emissionApr,
+  isSupply,
+}: ReserveAprParams) => {
+  const theme = useTheme();
+
+  return (
+    <Tooltip
+      title={
+        <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+          <Typography variant="body2">Amount earned/charged:</Typography>
+          <Typography variant="body2">
+            {`${reserveSymbol} ${`${formatter.toPercentage(reserveApr)}`}`}
+          </Typography>
+          {emissionApr && (
+            <Typography variant="body2">{`BLND ${formatter.toPercentage(emissionApr)}`}</Typography>
+          )}
+          <Typography variant="body2">
+            {`Net APR ${formatter.toPercentage(
+              isSupply ? (emissionApr ?? 0) + reserveApr : reserveApr - (emissionApr ?? 0)
+            )}`}
+          </Typography>
```

Comment:
We should separate this based on `isSupply`.

A suggestion on potentially clearer working:

For borrow:
```
XLM interest charged: 1.23%
BLND emissions earned: 2.34%
Net APR: -1.11%
```

For supply:
```
XLM interest earned: 1.23%
BLND emissions earned: 2.34%
Net APR: 3.57%
```